content/composer/overview.md

```diff
@@ -50,46 +89,27 @@ To connect networks, subnets and ports to a platform node, click and drag a line
 
 ![Blueprints List]({{< img "composer/connect-to-network.png" >}})
 
-### Adding Custom Node Types
-You can add custom node types by creating new ones, or by importing them. 
-
-**Creating a Custom Node Type**<br />
-You create custom node types on the **Definitions** tab.
-
-1  On the **Definitions** tab, click **New Type**. <br />
-2  Specify the settings for the new node type, including  where it  derives from, and any additional properties and interfaces.<br />
-3  Save the new node type.
-
-The new node type appears in the Custom Types list in the Stencils panel. You can use it in the same way as the built-in node types, by dragging and dropping it onto the canvas.
+### Adding Plugins to the Blueprint Package
 
-![Custom Node Types]({{< img "composer/custom-node-types.png" >}})
+Composer enables you to package plugins together with the .yaml file, so that they are part of the archive that you download or upload to Cloudify Manager. The recommended way to work with plugins is to upload them to the Manager, not to tie them with the blueprint package, therefore you would usually only use this option if you want to use plugins that you have written yourself, or if you have very specific reasons to package them with the blueprint itself.
 
-### Adding Plugins to the Blueprint Package
 **Adding a Plugin**<br />
 You add plugins on the **Definitions** tab. Cloudify supports many plugins, which you can [access here](http://getcloudify.org/downloads/plugin-packages.html). In addition, you can create your own plugins.
 
-**Creating a Custom Plugin**<br />
+**Adding a Plugin**<br />
 1  Click **Add Plugin** and specify the following properties:<br />
     - The plugin file name<br />
     - The Executor<br />
     - The URL or a local archive of the specified plugin  <br /><br>
   ![Create custom plugin]({{< img "composer/add-new-plugin.png" >}})
+  
 2  Click **Save** to save the properties that you have specified.<br />
 
-After a plugin is attached to a package, the operations it exposes appear in the interface’s operations implementations tree, as shown in the following screen capture.<br /> 
-
-![Implementations Tree]({{< img "composer/implementation-tree.png" >}})
+After a plugin is attached to a package, the operations it exposes appear in the interface’s operations implementations tree.
 
 ### Adding a Relationship Type
 Custom Relationships, like types, derive from existing relationships and can also have additional properties and interfaces. Interfaces are defined per the source and target nodes that define the relationship.
 
```

Comment:
We should add how to add a new relationship type here: 
To create custom relationships in your blueprint, go the “Relationship” tab in the stencils pane and choose the “+” sign. 
![screen shot 2017-08-06 at 5 29 26 pm](https://user-images.githubusercontent.com/19908069/29018003-0cf88658-7b62-11e7-9559-aebac9976f56.png)

You will now be able to give the Relationship the wanted properties and interfaces:
![screen shot 2017-08-06 at 5 30 26 pm](https://user-images.githubusercontent.com/19908069/29018069-44404f24-7b62-11e7-8409-b872f954f179.png)

(This also appears in the google doc but I forgot to mark it in yellow.